tests/base/geometric_utilities_02.cc

```diff
@@ -0,0 +1,71 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+// check GeometricUtilities::Coordinates::to_spherical and
+// GeometricUtilities::Coordinates::from_spherical.
+
+#include "../tests.h"
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/geometric_utilities.h>
+#include <deal.II/base/exceptions.h>
+
+#include <fstream>
+#include <cstdlib>
+
+using namespace dealii;
+using namespace dealii::GeometricUtilities::Coordinates;
```

Comment:
Why do you import the namespace `GeometricUtilities::Coordinates` here when you later access the functionality in lines 52 and 53 with the full namespace qualifier? In addition, you do not need to import the `dealii` namespace here because that is already done in `tests.h`.